### DISCUSSION TITLE
When adding directories, copilot messes it up.

### DISCUSSION THREAD

**Discussion by ieldante** [community/community#36903] [Copilot Conversations]
So I used copilot for about two months before it got released, and I had an issue with it. It might be fixed, and if so, I'm sorry. When I would type a directory in JS. (require(../../../util/test)) Copilot would insert what it thinks the directory should be. (I use vscode, and you can use TAB to insert a directory, copilot just auto-fills, and I have nothing to do about it).

Sorry if this is useless because I am not using it right, or it's fixed.

**Comment by ossanna16** [comment]
Hey there @DanteAndHubble and thank you for reporting this! I'm glad you found our community 🙂

**Comment by iamcjwilliams** [comment]
@DanteAndHubble Can you provide a screenshot of what you're experiencing?
